brigade/templates/project_showcase.html

```diff
@@ -0,0 +1,211 @@
+    </div>
+    <div class="width-one-third">
+      <div class="alert">
+        <div class="alert__icon">
+          <i class="fab fa-youtube"></i>
+        </div>
+        <div class="alert__content">
+          <strong>Want to learn more about food security projects in the Network?</strong><br> 
+          <a href="https://www.youtube.com/watch?v=jvVZHmMmq9I">Watch the Brigade Food Security Workshop video!</a>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="grid-box"><hr></div>
+  <div class="grid-box" id="transportation">
+    <div class="width-two-thirds">
+      <h2>Transportation</h2>
+      <div class="card">
+        <div class="card__body">
+          <h3 class="card__title">CyclePhilly</h3>
+          <p>CyclePhilly is a smartphone app for recording your bicycle trips. Data from the app can be used by regional transportation planners in the Philadelphia area to make Philly a better place to ride.</p>
+          <div class="list-group">
+            <div class="list-group__list-item">
+              <strong>CyclePhilly</strong> <span class="no-wrap">by <a href="/brigades/Code-for-Philly/">Code for Philly</a></span>
+              <div class="list-item__actions">
+                <a href="https://github.com/CodeForPhilly/cyclephilly" class="button" target="_blank">View code <i class="fab fa-github"></i></a>
+                <a href="http://www.cyclephilly.org/" class="button" target="_blank">Visit project <i class="fas fa-external-link-alt"></i></a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="grid-box"><hr></div>
+  <div class="grid-box" id="public-works">
+    <div class="width-two-thirds">
+      <h2>Public Works</h2>
+      <div class="card">
+        <div class="card__body">
+          <h3 class="card__title">Adopt a Drain</h3>
+          <p>Adopt a Drain Program enables residents to “adopt” a storm drain in their city or town, keeping it free of debris. While helping to reduce localized flooding, the program also fosters community engagement by encouraging residents to take an active role in improving their neighborhood.</p>
+          <div class="list-group">
+            <div class="list-group__list-item">
+              <strong>Adopt a Drain SF</strong> <span class="no-wrap">by <a href="/brigades/Code-for-San-Francisco/">Code for San Francisco</a></span>
+              <div class="list-item__actions">
+                <a href="https://github.com/sfbrigade/adopt-a-drain" class="button" target="_blank">View code <i class="fab fa-github"></i></a>
+                <a href="https://adoptadrain.sfwater.org/" class="button" target="_blank">Visit project <i class="fas fa-external-link-alt"></i></a>
+              </div>
+            </div>
+            <div class="list-group__list-item">
+              <strong>Norfolk Drains</strong> <span class="no-wrap">by <a href="/brigades/Code-for-Hampton-Roads/">Code for Hampton Roads</a></span>
```

Comment:
nit: these internal URLs should use `url_for('brigade.brigade', brigadeid='Code-for-Hampton-Roads')`

Reply:
Good catch! Will fix that.